modules/troubleshooting-network-observability-controller-manager-pod-out-of-memory.adoc

```diff
@@ -6,26 +6,41 @@
 [id="controller-manager-pod-runs-out-of-memory_{context}"]
 = Network Observability controller manager pod runs out of memory
 
-You can increase memory limits for the Network Observability operator by patching the Cluster Service Version (CSV), where Network Observability controller manager pod runs out of memory.
+You can increase memory limits for the Network Observability operator by editing the `spec.config.resources.limits.memory` configuration in the `Subscription` object.
 
 .Procedure
```

Comment:
As this is a troubleshooting procedure, some prereqs should exist for someone to reach a certain stage?